### PR TITLE
Apply: Francis feels frustrated post

### PR DESCRIPTION
**Pull request by MaeveRoseveare**
Although we did this work some time ago, we originally wrote this story in a PPT. We felt really proud of it and worried that it would just get lost (especially in the Google migration). So I finally found the time to create a design history out of it so more people can have access to it.

### CODE REVIEW

File: app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+Randomly, ExtrA Learning Trust asks for Francis’ bank details, she’s not sure why.  
+
+Francis finds out she will get £175 a week to do her SKE course.  
+
+It’s less than she earns as a waiter and she’s halfway through her SKE course already.  
+
+“Bit late, but ok”
+
+Francis fills out another form to provider her bak details.
```
Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md]:
```suggestion
Francis fills out another form to provider her bank details.
```

File: app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+ogImage:
+  src: francis-cover-image.png
+  alt: "Illustration showing a section called 'Interview availability in a task list. This screen then connects to the next screen with a question asking the user if there are any times they will not be available for interviews."
+---
+
+![Illustration showing a section called 'Interview availability in a task list. This screen then connects to the next screen with a question asking the user if there are any times they will not be available for interviews.](francis-cover-image.png)
+
+Previously, we designed guidance to tell candidates they have to do a [subject knowledge enhancement (SKE) course](/apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/) before their teacher training.
+
+We tested these designs and learnt a lot about SKE courses. Some of the findings we uncovered were about the wider SKE policy and do not fit with Apply for teacher training (Apply).
```
Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md]: I wonder if this opening para could be a bit more punchy, and summarise (or at least hint at) the issues themselves? Might also be good to briefly summarise what the research consisted of?

This suggestion isn’t quite right either, but might be an approach to try?

```suggestion
We did some research around subject knowledge enhancement (SKE) courses, as part of [designing guidance telling candidates when they have to do these](/apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/) before their teacher training.

We usability tested the designs with X participants, and also sent a survey to X people who had previously received an offer with a SKE course condition attached.

This research revealed some wider issues with how users choose a course, as well as the process of applying to a a course and receiving the funding, and the content of the course itself.
```

File: app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+We wanted to capture these findings in a way that would help people empathise with candidates and providers. So, we wrote a story....
+
+We’ve added the story as a design history to share with other teams. It's purpose is to think about different ways to present research and show empathy beyond individual services.
```
Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md]: I feel like this paragraph could be dropped?  The previous paragraph has a nice lead-in to the story already.

```suggestion
```

**Review by frankieroberto — COMMENTED** [review]
Love it! 😍

File: app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+ogImage:
+  src: francis-cover-image.png
+  alt: "Illustration of a woman with her hands on her head looking frustrated. Around her head are icons of exclamation marks and question marks showing her confusion. The the left of her it says 'When Francis felt frustrated."
+---
+
+![Illustration of a woman with her hands on her head looking frustrated. Around her head are icons of exclamation marks and question marks showing her confusion. The the left of her it says 'When Francis felt frustrated.](francis-cover-image.png)
+
+Previously, we designed guidance to tell candidates they have to do a [subject knowledge enhancement (SKE) course](/apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/) before their teacher training.
+
+We usability tested the designs with 7 participants. We also sent a survey and received 232 responses from people who had received an offer with a condition that they do an SKE course before their training started.
```
Comment (lsalivonaite-transform) [inline app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md]: Reconsider the first sentence here maybe about the 'usability tested'

File: app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+
+Francis makes some friends on her teacher training course. They also had to do an SKE course.  
+
+Everyone has different things to say about their SKE course – from different lengths to different content.  
+
+Some thought they were fabulous, some thought they were frightening.
+
+Francis loves her teacher training course, but she’s relearning things she did in her SKE course.  
+
+ Francis feels frustrated (again).
```
Comment (lsalivonaite-transform) [inline app/posts/apply-for-teacher-training/2023-07-12-when-francis-felt-frustrated.md:185]: Is there a solution to Francis problems?

**Review by lsalivonaite-transform — APPROVED** [review]

**Pull request merged**